tests/foreman/cli/test_subscription.py

```diff
@@ -260,8 +262,8 @@ def test_positive_delete_manifest_as_another_user(self):
         self.assertEquals(0, len(Subscription.list({'organization-id': org.id})))
 
     @tier2
-    @pytest.mark.usefixtures("ContentHostSetup")
-    def test_positive_Subscription_status_disabled(self):
+    @pytest.mark.usefixtures("GoldenTicketHostSetup")
+    def test_positive_subscription_status_disabled_golden_ticket(self):
```

Comment:
I was trying to figure out difference between this test and new one you add below, and... didn't we actually put API test into tests/foreman/**cli**/ directory?

This test is almost identical as `tests/foreman/api/test_subscription.py::test_positive_subscription_status_disabled`, the only difference is:
```diff
- vm.register_contenthost(self.org_setup.label, self.ak_setup.name)
+ vm.register_contenthost(self.org_setup['label'], self.ak_setup['name'])
```

Reply:
@mirekdlugosz I realized I used API Methods in the CLI test. However, after looking more into it, I do not think there is a way to get the "Subscription Status" value from the CLI. I have looked under Host() and Subscription(). any ideas?

Reply:
@ColeHiggins2 I played briefly with hammer and took a short look at Satellite code and I think you are right - hammer doesn't display "Subscription Status" value. There's section called "Subscription Information", but it doesn't contain this particular field.

Overall, if this scenario can't be covered by CLI, then it can't be covered by CLI. I would suggest to remove this particular test as duplicate.

You might try reaching out to dev / creating bugzilla to add this field to hammer output and see if that's something they could do in reasonable time (so it might make sense to `@stub` this test in meantime instead of removing it).